### PR TITLE
Adds original exception where possible.

### PR DESCRIPTION
**Pull request by goober**
Adds original exception to all `BankException` which are thrown from a catch block.
This will improve the traceability and make it possible to find the root cause of a thrown exception.

### CODE REVIEW

File: bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/AkeliusInvest.java [hunk 1]
```diff
@@ -209,11 +209,9 @@ public void updateTransactions(Account account, Urllib urlopen) throws LoginExce
...
 		} catch (ClientProtocolException e) {
-			// TODO Auto-generated catch block
-			e.printStackTrace();
+			throw new BankException(e.getMessage(), e);
 		} catch (IOException e) {
-			// TODO Auto-generated catch block
-			e.printStackTrace();
-		}
+            throw new BankException(e.getMessage(), e);
+        }
```
Comment (fredrike) [inline bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/AkeliusInvest.java]: Please watch the indentation..

Comment (goober) [inline bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/AkeliusInvest.java]: Could it be a github problem? The indentation looks fine in my IDE.

Comment (fredrike) [inline bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/AkeliusInvest.java]: No, you probably have tabs set to 2 spaces, git have 4 spaces for tabs..

We should try to get rid of all tabs and use 2 spaces instead (seems to be default setting in *Android Studio*)

**Comment by fredrike** [comment]
Thank you!

File: bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/AkeliusInvest.java (hunk 1 above)
Comment (goober) [inline bankdroid-legacy/src/main/java/com/liato/bankdroid/banking/banks/AkeliusInvest.java]: I will verify this. However, the formatting of all files should be done in a separate PR

**Pull request merged**